doc/releases/changelog-dev.md

```diff
@@ -47,6 +47,37 @@ keyword argument when using `GellMann`, which determines which of the 8 Gell-Man
         1: ─╰RXY(1.570796+0.00j)─╰RYX(1.570796+0.00j)─┤ ╰<Z@Z>
   ```
 
+* Added the `qml.map_wires` function, that changes the wires of the given operator, `QNode`, queue
+  or quantum function according to the given wire map.
+  [(#3145)](https://github.com/PennyLaneAI/pennylane/pull/3145)
+
+  Using `qml.map_wires` with an operator:
+
+  ```pycon
+  >>> op = qml.RX(0.54, wires=0) + qml.PauliX(1) + (qml.PauliZ(2) @ qml.RY(1.23, wires=3))
+  >>> op
+  (RX(0.54, wires=[0]) + PauliX(wires=[1])) + (PauliZ(wires=[2]) @ RY(1.23, wires=[3]))
+  >>> wire_map = {0: 3, 1: 2, 2: 1, 3: 0}
+  >>> qml.map_wires(op, wire_map)
+  (RX(0.54, wires=[3]) + PauliX(wires=[2])) + (PauliZ(wires=[1]) @ RY(1.23, wires=[0]))
+  ```
+
+  Using `qml.map_wires` with a `QNode`:
+
+  ```pycon
+  >>> dev = qml.device("default.qubit", wires=4)
+  >>> @qml.qnode(dev)
+      def circuit():
+          qml.RX(0.54, wires=0) @ qml.PauliX(1) @ qml.PauliZ(2) @ qml.RY(1.23, wires=3)
+          return qml.probs(wires=0)
```

Comment:
```suggestion
  >>> @qml.qnode(dev)
  ... def circuit():
  ...     qml.RX(0.54, wires=0) @ qml.PauliX(1) @ qml.PauliZ(2) @ qml.RY(1.23, wires=3)
  ...     return qml.probs(wires=0)
```

Reply:
Actually, for these examples, it's probably best to:
* Use plain ops, not operator arithmetic
* Show the circuit using `qml.draw`, not exposing the `tape`
* Use an obvious wire map, like `{0:10, 1:11, 2:12, 3:13}` or `{0:"a", 1:"b", 2:"c", 3:"d"}`

Reply:
> - Use plain ops, not operator arithmetic
Done
> - Show the circuit using qml.draw, not exposing the tape
If we do this the wire values are not shown:
```pycon
>>> print(qml.draw(circuit)())
0: ──RX(0.54)─┤  Probs
1: ──X────────┤
2: ──Z────────┤
3: ──RY(1.23)─┤
```
> - Use an obvious wire map, like {0:10, 1:11, 2:12, 3:13} or {0:"a", 1:"b", 2:"c", 3:"d"}
Using {0:10, 1:11, 2:12, 3:13} ✅

Reply:
`circuit` was the unmapped qnode:

```pycon
>>> print(qml.draw(mapped_circuit)())
10: ──RX(0.54)─┤  Probs
11: ──X────────┤       
12: ──Z────────┤       
13: ──RY(1.23)─┤       
```

Had me concerned for a moment.

Reply:
Shoot, sorry for that 😢

Reply:
Changed!